src/components/Cart.tsx

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable max-len */
+import { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { getFullPrice } from '../helpers/getFullPrice';
+import { useLocalStorage } from '../helpers/useLocalStorage';
+import { Product } from '../types/Product';
+import { CartItem } from './CartItem';
+
+const getTotalPrice = (products: Product[]) => {
+  let result = 0;
+
+  for (let i = 0; i < products.length; i += 1) {
+    result += getFullPrice(products[i].price, products[i].discount) * products[i].number;
+  }
+
+  return result;
```

Comment:
```suggestion
  return products.reduce((result, product) => result + getFullPrice(product.price, product.discount) * product.number);
```

Reply:
That's good advice :arrow_up: 
Why you didn't do it?

It's always better to use Array methods than common loop